pywren_ibm_cloud/executor.py

```diff
@@ -461,6 +460,8 @@ def clean(self, fs=None, cs=None, cloudobjects=True, force=True, log=True):
         Deletes all the files from COS. These files include the function,
         the data serialization and the function invocation results.
         """
+        self.invoker.cleanup() 
```

Comment:
@kpavel this step deletes all data from COS. in many flows, we will need to stop VM but keep data in COS